fr/translation.json

```diff
@@ -476,5 +476,10 @@
   "oldest_backlog_tx_age": "Âge de transaction le plus ancien dans le backlog",
   "tx_time_accepted_l2": "Temps estimé pour qu'une transaction soit acceptée sur le bloc L2",
   "cairo_book": "Le livre du Caire",
-  "estimated_tx_time": "Temps de transaction estimé"
+  "estimated_tx_time": "Temps de transaction estimé",
+  "active_users": "Utilisateurs actifs",
+  "protocol_leaderboard": "Classement du protocole",
```

Comment:
Is there only one protocol in the leaderboard?
If there's multiple ones, change to `Classement des protocoles`

Reply:
yes there are multiple protocols let me update